parm/convection_permitting/namelist.atmosphere

```diff
@@ -53,28 +53,14 @@
     config_radtlw_interval = '00:15:00'
     config_radtsw_interval = '00:15:00'
     config_bucket_update = 'none'
-    config_physics_suite = '${physics_suite}'
-    config_microp_scheme = 'mp_thompson'
-    config_thompson_hail_aware = true
-    config_convection_scheme = 'off'
+    ! config_physics_suite = '${physics_suite}'
+    config_physics_suite = 'hrrrv5'
```

Comment:
1. We need to use line 56 and define correct `physics_suite` in `exrrfs_fcst.sh`
2. To confirm, do we have a `hrrrv5` physics suite ready at this moment?

Reply:
This is the namelist that works for the current GSL MPAS model, consistent with the GSL real-time runs. My previous tests using "convection_permitting" would crash. For details we need inputs from the physics people.

Reply:
@chunhuazhou Thanks for the reply. We better NOT hardcode this to "hrrrv5", we can generate this on-the-fly in the `exrrfs-fcst.sh` script. So remove line 57, revert line 56, and specify "hrrrv5" in the script.

For the physics suite, I did not run the new GSL version of MPAS, so we will need @hu5970's input on this.